### ISSUE TITLE
Upgrades: Verification of control plane succeeded even though component was down

### ISSUE THREAD

**Issue by alexbrand** [apprenda/kismatic#300]
The play that verifies the control plane did not catch the fact that the scheduler was in a crash loop:

```
root@node001:~# kubectl get pods -n kube-system
NAME                                    READY     STATUS             RESTARTS   AGE
calico-node-x2w86                       2/2       Running            0          13m
default-http-backend-85dg1              1/1       Running            2          20m
ingress-jm4h6                           1/1       Running            1          20m
kube-apiserver-node001                  1/1       Running            0          15m
kube-controller-manager-node001         1/1       Running            0          15m
kube-dns-5d9x3                          3/3       Running            0          13m
kube-dns-v19-896g0                      3/3       Running            3          20m
kube-proxy-node001                      1/1       Running            0          14m
kube-scheduler-node001                  0/1       CrashLoopBackOff   7          15m
kubernetes-dashboard-1389325151-9xvn7   1/1       Running            0          12m
```

Upon further investigation, I noticed that the systemd kube-scheduler service was not shut down, which prevented the self-hosted scheduler to start up (due to port conflicts).

**Comment by alexbrand** [comment]
There is some kind of intermittent bug in K8s where the scheduler dies after loosing connection to the API server: https://github.com/kubernetes/kubernetes/issues/41428. This results in systemd restarting the service after 3 seconds, which is racing against our own `systemctl stop kube-scheduler`.

**Comment by alexbrand** [comment]
Fixed in #301
